src/Components/CreateImageWizard/formComponents/ActivationKeys.js

```diff
@@ -80,7 +80,7 @@ const ActivationKeys = ({ label, isRequired, ...props }) => {
           isDisabled={!isSuccessActivationKeys}
         >
           {isSuccessActivationKeys &&
-            activationKeys.body.map((key, index) => (
```

Comment:
I don't think that we can remove the body property, it is part of the API response:
https://console.redhat.com/docs/api/rhsm/v2#operations-activationKey-listActivationKeys

I made a mistake when I was adding typing to this endpoint. I just learned that there is an official tool for generating RTKQ query hooks from OpenAPI specifications. I'm going to try that and see how it works, then revisit this PR:
https://redux-toolkit.js.org/rtk-query/usage/code-generation